faust-state/src/lib.rs

```diff
@@ -452,22 +439,21 @@ impl ParamsBuilder {
     ) {
         let prefix = self.prefix[..].join("/").to_string();
         let idx = idx.0;
-        if self.inner.contains_key(&idx) {
-            let node = self.inner.get_mut(&idx).unwrap();
-            node.label = label.to_string();
-            node.typ = typ;
-            if let Some(mut metadata) = metadata {
-                node.metadata.append(metadata.as_mut());
-            }
-        } else {
+        if let std::collections::hash_map::Entry::Vacant(e) = self.inner.entry(idx) {
```

Comment:
Oh nice! Didn't know about this

Reply:
this is one of those things clippy knows for us :)

Reply:
clippy is such a good boy!